cloudwatch_metrics_collector/include/cloudwatch_metrics_collector/metrics_collector_parameter_helper.hpp

```diff
@@ -48,13 +53,28 @@ constexpr char kNodeParamMaximumFileSize[] = "maximum_file_size";
 constexpr char kNodeParamStorageDirectory[] = "storage_directory";
 constexpr char kNodeParamStorageLimit[] = "storage_limit";
 
+enum class TopicType
```

Comment:
Thomas suggested reading two lists (https://github.com/aws-robotics/cloudwatchmetrics-ros2/pull/14#discussion_r371374483).

I think it's easier for a user to write a config file that has two lists than a config file that has a map.

Reply:
> Thomas suggested reading two lists ([#14 (comment)](https://github.com/aws-robotics/cloudwatchmetrics-ros2/pull/14#discussion_r371374483)).
> 
> I think it's easier for a user to write a config file that has two lists than a config file that has a map.

Sounds good, I like @thomas-moulard's suggestion.

Reply:
Although I've changed the way the config file should be written, I've decided to keep the internal representation this way. This "boilerplate" provides more clarity than passing multiple vectors or a map of vectors or a vector of vectors around.

Reply:
How is implementing / maintaining the "boilerplate" less complex than maintaining two lists read directly from the config file? ```SubscribeAllTopics``` would become much simpler and clearer.

Reply:
All the function signatures will need to take an additional vector parameter and the `MetricsCollector` will need to have an additional vector member variable. It's just moving the "boilerplate" from this one place to be splattered all over many places. And it's also less scalable if you were to add yet another message type, where it would be adding another enum versus adding a third vector parameter to every function that gets passed topic vectors.